atd-vzd/migrations/default/1715960020005_crashes_list_and_fatality_metrics_views/up.sql

```diff
@@ -483,3 +487,120 @@ create view locations_list_view as (
     where locations.council_district > 0 and locations.location_group = 1
 );
 
+create or replace view location_crashes_view as (select
+    public.crashes.crash_id,
+    'CR3'::text as type,
+    public.crashes.location_id,
+    public.crashes.case_id,
+    to_char(
+        public.crashes.crash_timestamp at time zone 'US/Central', 'YYYY-MM-DD'
+    ) as crash_date,
+    to_char(
+        public.crashes.crash_timestamp at time zone 'US/Central', 'HH24:MI:SS'
+    ) as crash_time,
+    upper(
+        to_char(
+            public.crashes.crash_timestamp at time zone 'US/Central', 'dy'
+        )
+    ) as day_of_week,
+    crash_injury_metrics_view.crash_injry_sev_id as crash_sev_id,
+    public.crashes.latitude,
+    public.crashes.longitude,
+    public.crashes.address_primary,
+    public.crashes.address_secondary,
+    crash_injury_metrics_view.non_injry_count,
+    crash_injury_metrics_view.nonincap_injry_count,
+    crash_injury_metrics_view.poss_injry_count,
+    crash_injury_metrics_view.sus_serious_injry_count,
+    crash_injury_metrics_view.tot_injry_count,
+    crash_injury_metrics_view.unkn_injry_count,
+    crash_injury_metrics_view.vz_fatality_count,
+    crash_injury_metrics_view.est_comp_cost_crash_based,
+    lookups.collsn_lkp.label as collsn_desc,
+    crash_units.movement_desc,
```

Comment:
just wanted to note that I couldn't find an example of a row with this value populated in the test data so i updated a unit in crash `19478437` to check that this works. It works! ✅

Reply:
thanks for this!